testsuite/integration/web/pom.xml

```diff
@@ -1288,18 +1288,14 @@
                                         <arquillian.xml>arquillian-bootable.xml</arquillian.xml>
                                     </systemPropertyVariables>
                                     <excludes>
-                                        <!--TODO re-evaluate these exclusions from EE 9 work-->
                                         <!-- requires legacy-security -->
                                         <exclude>org/jboss/as/test/integration/web/security/**/*TestCase.java</exclude>
```

Comment:
@istudens @fjuma This seems wrong. Not just here or elsewhere related to ts.preview or ts.bootable, but perhaps anywhere in this pom.  We don't have legacy security anywhere any more and these tests run and pass on some jobs.

Of course it's possible some need to be excluded from some profiles for other reasons, e.g. they require adding a module etc.

Reply:
@bstansberry That's correct and I missed this. There is no legacy security even in bootable jar profiles, I've changed the PR accordingly. Thanks.